docs/Gemfile

```diff
@@ -0,0 +1,34 @@
+# © 2020 and later: Unicode, Inc. and others.
+# License & terms of use: http://www.unicode.org/copyright.html
+
+source "https://rubygems.org"
+# Hello! This is where you manage which Jekyll version is used to run.
+# When you want to use a different version, change it below, save the
+# file and run `bundle install`. Run Jekyll with `bundle exec`, like so:
+#
+#     bundle exec jekyll serve
+#
+# This will help ensure the proper Jekyll version is running.
+# Happy Jekylling!
+gem "jekyll", "~> 3.8.7"
+# This is the default theme for new Jekyll sites. You may change this to anything you like.
+gem "minima", "~> 2.5"
+# If you want to use GitHub Pages, remove the "gem "jekyll"" above and
```

Comment:
I don't have a local setup for building Jekyll, @echeran do you think you could perhaps make this change and push another commit? :)

Reply:
ping

Reply:
Removing that line works after testing using a local Jekyll instance.  I created a PR against @jefgen 's fork.  I think this might explain some of the confusion during the Jekyll setup exploratory work, I'll update the doc accordingly.

Reply:
Thank you @echeran!

Reply:
I've merged in the changes from @echeran, and pushed one more commit to tweak the comment wording slightly. Thanks again Elango!